k8s-deploy/docs/KubeFATE _API_Reference_Swagger.md

```diff
@@ -0,0 +1,18 @@
+# KubeFATE  API Reference Swagger
+
+## How to view swagger page?
+
+### swagger.io
+
+Copy the swagger.yaml file to https://editor.swagger.io/ to view.
+
+### local
+
+Deploy KubeFATE
+
+Open the URL http://kubefate.net/v1/swagger/index.html
+
+
+
+*Tip: The swagger.yaml file is generated by `swag init`.*
```

Comment:
*Tip: The swagger.yaml file is generated by `swag init`.* -> **Note: The swagger.yaml file is generated by `swag init`.**